src/tests/ftest/container/cont_rf_oclass_enforcement.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2019-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from container_rf_test_base import RbldContRedundancyFactor
+
+
+class RbldContRedundancyFactorEnforce(RbldContRedundancyFactor):
```

Comment:
That's the name that will show up in the test?

Reply:
Have shorten the class name. please see the new commit.